tools/cldr-apps/.settings/org.eclipse.wst.common.component

```diff
@@ -1,35 +1,48 @@
 <?xml version="1.0" encoding="UTF-8"?><project-modules id="moduleCoreId" project-version="1.5.0">
-            
+                
+    
     
     <wb-module deploy-name="cldr-apps">
-                        
+        <wb-resource deploy-path="/" source-path="/target/m2e-wtp/web-resources"/>
+                                
+        
+        
+        <wb-resource deploy-path="/" source-path="/src/main/webapp" tag="defaultRootSource"/>
+                                
         
-        <wb-resource deploy-path="/" source-path="/src/main/webapp"/>
-                        
         
         <wb-resource deploy-path="/WEB-INF/lib" source-path="/WebContent/WEB-INF/lib"/>
-                 
+                         
+        
         
         <!-- needed? -->
-                        
+                                
+        
         
         <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/main/java"/>
-                        
+                                
+        
         
         <wb-resource deploy-path="/WEB-INF/classes" source-path="/.apt_generated"/>
                 
-        <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/test/java"/>
-        <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/test/resources"/>
         <wb-resource deploy-path="/WEB-INF/classes" source-path="/src/main/resources"/>
-                        
+        <wb-resource deploy-path="/WEB-INF/classes" source-path="/.apt_generated_tests"/>
+        <dependent-module archiveName="cldr-39.0-SNAPSHOT.jar" deploy-path="/WEB-INF/lib" handle="module:/resource/cldr/cldr">
```

Comment:
How will this dependency on the version -- here 39.0 -- be handled? Is this set within the Eclipse interface, or ...?

Reply:
this should get automatically updated by eclipse.